### PR TITLE
feat: scroll to bottom when adding a new native filter and the page is filled

### PR DESCRIPTION
**Pull request by diegomedina248**
### SUMMARY

When adding a native filter, and the page is already filled (with other filters), auto-scroll to the bottom so that the newly added filter is visible on the left panel.

Also, fixes a typo in the onRearra**n**ge function.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:

https://user-images.githubusercontent.com/17252075/157144489-9c4b5eef-f1b9-41cb-bca5-d9a7c9a5bac1.mov

After:

https://user-images.githubusercontent.com/17252075/157144137-e2a3e265-97eb-40fb-a18e-23a8d6386951.mov

### TESTING INSTRUCTIONS

 1. Go to any dashboard
 2. Hit the Add/Edit filters button on the left panel
 3. Start adding filters & dividers
 4. Ensure that, when the filter container fills the screen, adding new filters scrolls the container to the bottom

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by rusackas** [comment]
Unit tests have failed on this one a couple times despite re-running CI... could you check to see if they're passing locally?

File: superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FilterConfigPane.test.tsx [hunk 1]
```diff
@@ -65,7 +72,7 @@ test('renders form', async () => {
 
 test('drag and drop', async () => {
   defaultRender();
-  // Drag the state and contry filter above the product filter
+  // Drag the state and country filter above the product filter
```
Comment (rusackas) [inline superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FilterConfigPane.test.tsx:75]: Thanks for cleaning up all the typos and things along the way. You're making the world a better place! 🧹 ✨

**Review by rusackas — APPROVED** [review]
LGTM, and works well in testing. Just re-triggered E2E tests 🤞 . CC @kgabryje and @michael-s-molina in case you spot any risks I didn't.

**Review by michael-s-molina — APPROVED** [review]

**Review by kgabryje — APPROVED** [review]
LGTM

**Comment by sadpandajoe** [comment]
🏷️ preset:2022.11

**Pull request merged**